cmd/note_common.go

```diff
@@ -164,44 +170,65 @@
 			} else {
 				oldfile = f[4]
 			}
-		} else if strings.HasPrefix(scanner.Text(), "|") {
-			if comments != "" {
-				createCommitNote(project, mrID, commit, newfile, oldfile, oldLineNum, newLineNum, comments, block)
-				comments = ""
+			continue
+		}
+
+		// read line numbers
+		f := strings.Fields(strings.TrimSpace(scanner.Text()))
+		if f[1] == "@@" {
+			continue
+		}
+
+		oldLineNum = -1
+		newLineNum = -1
+
+		// The output can be, for example:
+		// | # # [no +/-] < context comment
```

Comment:
there's another bug: if a context line starts with - or +, we interpret it as - or + line:

	| newfile: README.md oldfile: README.md
	|      @@ -1,5 +1,6 @@
	| 1  1  - context line
	|    2 +- added line 

Probably shouldn't be using strings.Fields but a regex like `^\|(?: +(\d+)){1,2} ([-+ ])`

Reply:
Good point.  I'm going to fix this up in a follow-up commit.  I think the same bug was there before this change.